internal/storage/sql_provider.go

```
@@ -1335,3 +1363,74 @@
+
+	switch {
+	case errors.Is(err, sql.ErrNoRows):
+		return model.User{}, errors.New(errUserNotFound)
+	case err != nil:
+		return model.User{}, fmt.Errorf(errLoadingUserDetails, identifier, err)
+	}
+	if user.Password, err = p.decrypt(user.Password); err != nil {
+		p.log.WithError(err).WithField("user", identifier).
+			Warning("Failed to decrypt user password, marking account for password reset")
+		user.Password = []byte{}
+		// TODO: Consider setting a flag to force password reset
```

golangci-lint (1.62.2)

Comment should end in a period

(godot)